src/Spinner/Spinner.scss

```diff
@@ -3,45 +3,21 @@
     justify-content: center;
     align-items: center;
     text-align: center;
-    min-height: 100vh;
-    
+    width: 100%;
+    padding: 1em;
   }
 
   .pokeball{
```

Comment:
What happens to the other spinner classes?. where they removed for any reason?

Reply:
The pokeball class has changed since now we are using an image for the spinner and not a pure css styled div, in order to simplify scaling